src/routes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/Liitteet.js

```diff
@@ -0,0 +1,257 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import { COLORS, TRANSITIONS } from "../../../../../../modules/styles"
```

Comment:
Lintterillä on paljon sanottavaa tästä tiedostosta. Meillä ei toistaiseksi ole yhteisiä tyylikäytäntöjä, mutta kun kerran ajetaan ESLintiä kehitysympäristössä, kai sitä kannattaisi seuratakin. ;)

Reply:
Juu, viimeistelen näitä vielä nyt. Katsoin jo yhdesti, mutta jäi vielä.